docs/maintaining/tasks.md

```diff
@@ -26,20 +20,20 @@ update the pinned SHA in the
 Submit it as a regular PR and go through the build process. Expect the build to take a while to
 complete: every image layer will rebuild.
 
-## Adding a New Core Image to Docker Cloud
+## Adding a New Core Image to Docker Hub
 
 When there's a new stack definition, do the following before merging the PR with the new stack:
 
 1. Ensure the PR includes an update to the stack overview diagram
    [in the documentation](https://github.com/jupyter/docker-stacks/blob/master/docs/using/selecting.md#image-relationships).
    The image links to the [blockdiag source](http://interactive.blockdiag.com/) used to create it.
 2. Ensure the PR updates the Makefile which is used to build the stacks in order on GitHub Actions.
-3. Create a new repository in the `jupyter` org on Docker Cloud named after the stack folder in the
+3. Create a new repository in the `jupyter` org on Docker Hub named after the stack folder in the
    git repo.
 4. Grant the `stacks` team permission to write to the repo.
 5. Click _Builds_ and then _Configure Automated Builds_ for the repository.
 6. Select `jupyter/docker-stacks` as the source repository.
-7. Choose _Build on Docker Cloud's infrastructure using a Small node_ unless you have reason to
+7. Choose _Build on Docker Hub's infrastructure using a Small node_ unless you have reason to
```

Comment:
I think sections from 5 to 12 are outdated.
Can I safely remove them?

Reply:
I don't think we have any `Automated Builds` in Docker Hub now.

Reply:
I will delete these lines, but this have to be checked.

Reply:
I think you are correct - delete away!

Reply:
Done, with improvement to the current status of the project.